k256/src/ecdsa.rs

```diff
@@ -154,11 +154,20 @@ use crate::Secp256k1;
 
 #[cfg(feature = "ecdsa")]
 use {
-    crate::{AffinePoint, FieldBytes, Scalar},
-    ecdsa_core::hazmat::{SignPrimitive, VerifyPrimitive},
-    elliptic_curve::{ops::Invert, scalar::IsHigh, subtle::CtOption},
+    crate::elliptic_curve::bigint::CheckedAdd,
+    crate::elliptic_curve::ops::Reduce,
+    crate::elliptic_curve::point::DecompressPoint,
+    crate::elliptic_curve::{
+        ops::Invert, scalar::IsHigh, subtle::CtOption, Curve, CurveArithmetic, FieldBytesEncoding,
+        PrimeField,
+    },
+    crate::ORDER,
+    crate::{arithmetic::scalar::Scalar, AffinePoint, FieldBytes, PowdrAffinePoint},
+    ecdsa_core::hazmat::{bits2field, SignPrimitive, VerifyPrimitive},
 };
 
+type U = <Secp256k1 as Curve>::Uint;
```

Comment:
```suggestion
type Uint = <Secp256k1 as Curve>::Uint;
```
`U` looks like a generic parameter